Admin/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="./admin.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>Admin page</header>
+    <main>
+        <section>
+            <a href="../index.html"><button>back to home page</button></a>
```

Comment:
across your entire site, you'll want to implement links to your cart/admin page/ home(products).  I have no way of getting to your form from your deployed site at the moment